project_national/fact_sheets_category_1.yml

```diff
@@ -0,0 +1,466 @@
+        lifetime: &lifetime_ashp_ducted_SEER_15_9_HSPF 15
+      # MSHP
+      - &heat_pump_fed_min_ductless_southern
+        option: HVAC Heating Efficiency|MSHP, SEER 15, 8.8 HSPF
+        apply_logic: &logic_fed_min_heat_pump_MSHP_ductless_southern
+          - *logic_southern_state_federal_minimum
+          - HVAC Has Ducts|No
+          - &logic_not_MSHP_greater_than_8_8_HSPF
+            or:
+              - HVAC Heating Efficiency|MSHP, SEER 14.5, 8.2 HSPF
+        costs: &costs_mshp_ductless_SEER_15_9_HSPF
+        # Source: Custom regression by Brennan Less. Ductless heat pump project costs were regressed on nameplate tons,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ductless MSHP n=173, HSPF median 11, range 9.3–14.2. Implicitly includes multiple zones and electrical upgrades.
+        # Regression results in costs that are 15–30% higher than costs from The Heat Pump Store as documented in https://redwoodenergy.net/wp-content/uploads/2021/02/Pocket-Guide-to-All-Electric-Retrofits-of-Single-Family-Homes.pdf 
+        # LBNL data were not sufficient to include a relationship between SEER or HSPF and cost, so cost deltas for higher 
+        # and lower HSPF were added using the relationship between HSPF and cost for ductless MSHPs in REMDB (https://remdb.nrel.gov/).
+          - value: 2472.60
+            multiplier: Fixed (1)
+          - value: 320.42
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_mshp_ductless_SEER_15_9_HSPF 
+          15
+
+      # northern states
+      - &heat_pump_fed_min_ducted_northern
+        option: HVAC Heating Efficiency|ASHP, SEER 14, 8.8 HSPF # same as base option
+        apply_logic: &logic_fed_min_heat_pump_ASHP_ducted_northern
+          - *logic_northern_state_federal_minimum
+          - HVAC Has Ducts|Yes
+          - *logic_not_ASHP_greater_than_8_8_HSPF
+        costs: *costs_ashp_ducted_SEER_15_9_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_15_9_HSPF
+
+      # MSHP
+      - &heat_pump_fed_min_ductless_northern
+        option: HVAC Heating Efficiency|MSHP, SEER 14, 8.8 HSPF
+        apply_logic: &logic_fed_min_heat_pump_MSHP_ductless_northern
+          - *logic_northern_state_federal_minimum
+          - HVAC Has Ducts|No
+          - *logic_not_MSHP_greater_than_8_8_HSPF
+        costs: *costs_mshp_ductless_SEER_15_9_HSPF
+        lifetime: *lifetime_mshp_ductless_SEER_15_9_HSPF 
+      #global
+      - &heat_pump_ducted_cooling
+        option: HVAC Cooling Efficiency|Ducted Heat Pump # Applied regardless of which option is applied
+        apply_logic:
+          or:
+            - *logic_fed_min_heat_pump_ASHP_ducted_northern
+            - *logic_fed_min_heat_pump_ASHP_ducted_southern
+
+      - &heat_pump_ductless_cooling
+        option: HVAC Cooling Efficiency|Non-Ducted Heat Pump # Applied regardless of which option is applied
```

Comment:
This is a good checksum/failsafe, but it should not actually change any inputs, since this option is already only being applied to models with baseline MSHP.